queue/src/fiskaltrust.Middleware.Localization.QueueDEFAULT/Factories/SignatureItemFactoryDEFAULT.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using fiskaltrust.ifPOS.v1;
+using fiskaltrust.ifPOS.v1.it;
+using fiskaltrust.Middleware.Contracts.Factories;
+
+namespace fiskaltrust.Middleware.Localization.QueueDEFAULT.Factories
+{
+    public class SignatureItemFactoryDEFAULT : SignatureItemFactory
+    {
+        public override long CountryBaseState => 000000000;
+
+        public SignatureItemFactoryDEFAULT() { 
+        }
+
+        protected static NumberFormatInfo CurrencyFormatter = new ()
+        {
+            NumberDecimalSeparator = ",",
```

Comment:
Not sure about this property, but if we are talking about 10.0 we should probably use the international format for this and choose a .